### ISSUE TITLE
Tip Tweets Should Link to Recipient

### ISSUE THREAD

**Issue by jonathansampson** [brave/brave-browser#7698]
## Description 
Presently, if I tip a user on Twitch, I'll get a message like the following:

> I just tipped kitboga using the Brave Browser. Check it out at https://brave.com/tips. #TipWithBrave

This message shares a recipient name, but without a platform context, the name is not very helpful. It would be far better to link to the recipient's account on each platform:

> I just tipped twitch.tv/kitboga using the Brave Browser. Check it out at https://brave.com/tips. #TipWithBrave

We should support custom username links for each platform we support. One exception is Twitter, where we simply provide the user's name (complete with the @ prefix), which is automatically interpreted as a user link on the platform.

**Comment by bsclifton** [comment]
@jonathansampson is this the same as https://github.com/brave/brave-browser/issues/7698 ?

**Comment by jonathansampson** [comment]
@bsclifton I think you might have meant to link a different issue. You linked to this one 🙂

**Comment by bsclifton** [comment]
@jonathansampson doh - meant to link to https://github.com/brave/brave-browser/issues/7652 😄

**Comment by jonathansampson** [comment]
@bsclifton I thought you might have meant to link to that. These aren't the same issue, though they are similar. When you tip a tweet on Twitter, the message we generate includes a clickable username, as well as a reference to the Tweet you tipped. That isn't the case when you tip content on other sites. I'm drawing attention to the lack of a clickable name (the recipient, at the start of the tweet), and @lukemulks is drawing attention to the lack of context (the action, at the end of the tweet).

Example: I just tipped your previous comment, and this is the Tweet we generate:

> I just tipped Brian Clifton using the Brave Browser. Check it out at https://brave.com/tips. #TipWithBrave

Note that 'Brian Clifton' is not a clickable link (my issue). Also, no reference to the comment I tipped (Luke's issue). Contrast that with a Tweet from Scott Hanselman that I recently tipped:

> I just tipped @shanselman using the Brave Browser. Check it out at https://brave.com/tips. #TipWithBrave https://twitter.com/shanselman/status/1216747475307884550

Note we have both a clickable name, and a reference to the Tweet that I tipped.
